services/web-app/src/app/layout.tsx

```diff
@@ -1,16 +1,16 @@
 import "../styles/globals.css";
+import { Theme } from "@radix-ui/themes";
 import React, { ReactNode } from "react";
+
 import { NextAuthProvider } from "./providers";
-import { Footer } from "@/components/footer/footer";
-import "@radix-ui/themes/styles.css";
-import { Theme } from "@radix-ui/themes";
-import { NavBar } from "@/components/navbar";
+import Footer from "../components/footer/footer";
+import { NavBar } from "../components/navbar";
```

Comment:
[Note] I changed all imports to relative imports so we had consistency across the code base. In the future it would be nice to use absolute paths everywhere